### PR TITLE
adding solution to slicing lab exercises from lesson 3. 

### PR DESCRIPTION
**Pull request by elnurad**
I also combined the three parts of my solution to grid printer exercise into one file so I am adding it too.

### CODE REVIEW

File: students/eldonab/lesson03/strformat_lab.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+
+# Task three
+# Rewrite: "the 3 numbers are: {:d}, {:d}, {:d}".format(1,2,3)" to take an arbitrary number of values.
+
+def formatter(t):
+    l = len(t)
+    listed = " ,".join(["{}"] * l)
+    my_str = "The numbers are: " + listed
+    return (my_str).format(*t)
```
Comment (charlesrobison) [inline students/eldonab/lesson03/strformat_lab.py]: You'll want to include the count of numbers as well, as in the above note "the 3 numbers are...", etc.  You can use len() for that.

File: students/eldonab/lesson2/grid_printer_combined.py [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    print(" +")
+
+
+
+
+
+
+
+grid(3, 4)
+grid(5, 8)
```
Comment (charlesrobison) [inline students/eldonab/lesson2/grid_printer_combined.py]: Works well!

File: students/eldonab/lesson3/slicing_lab.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+#Assert statements to test our reversed function: checks whether the function returns the given argument with its elements reversed.elements
+assert reversed(a_string) == "gnirts a si siht", "assert statement failed"
+assert reversed(a_tuple) == (32, 5, 12, 13, 54, 2), "assert statement failed"
+assert reversed(a_list) == [16, 15, 14, 13, 12, 11, 10, 9, 8, 7, 6, 5, 4, 3, 2, 1], "assert statement failed"
+
+#Assert statements to test our third function: checks whether it returns the provided argument with 
+#the middle third, then last third and then the first third in the new order.the
+assert third(a_string) == "is a stringthis ", "assert statement failed"  
+assert third(a_tuple) == (13, 12, 5, 32, 2, 54), "assert statement failed"
+assert third(a_list) == [6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 1, 2, 3, 4, 5], "assert statement failed"
```
Comment (charlesrobison) [inline students/eldonab/lesson3/slicing_lab.py]: Looks good - nice assert statements!

**Comment by elnurad** [comment]
Thank you!

**Pull request merged**